src/main/java/com/redhat/cloud/notifications/routers/InternalService.java

```diff
@@ -0,0 +1,145 @@
+package com.redhat.cloud.notifications.routers;
+
+import com.redhat.cloud.notifications.db.ApplicationResources;
+import com.redhat.cloud.notifications.db.BehaviorGroupResources;
+import com.redhat.cloud.notifications.db.BundleResources;
+import com.redhat.cloud.notifications.models.Application;
+import com.redhat.cloud.notifications.models.Bundle;
+import com.redhat.cloud.notifications.models.EventType;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+
+import javax.inject.Inject;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.NotFoundException;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.Response;
+import java.util.List;
+import java.util.UUID;
+
+@Path("/internal")
+public class InternalService {
+
+    @Inject
+    BundleResources bundleResources;
+
+    @Inject
+    BehaviorGroupResources behaviorGroupResources;
+
+    @Inject
+    ApplicationResources appResources;
+
+    @POST
+    @Path("/bundles")
+    public Uni<Bundle> createBundle(@NotNull @Valid Bundle bundle) {
+        return bundleResources.createBundle(bundle);
+    }
+
+    @GET
+    @Path("/bundles")
+    public Uni<List<Bundle>> getBundles() {
+        // Return configured with types?
+        return bundleResources.getBundles();
+    }
+
+    @GET
+    @Path("/bundles/{bundleId}")
+    public Uni<Bundle> getBundle(@PathParam("bundleId") UUID bundleId) {
+        return bundleResources.getBundle(bundleId)
+                .onItem().ifNull().failWith(new NotFoundException());
+    }
+
+    @PUT
+    @Path("/bundles/{bundleId}")
+    public Uni<Response> updateBundle(@PathParam("bundleId") UUID bundleId, @NotNull @Valid Bundle bundle) {
+        return bundleResources.updateBundle(bundleId, bundle)
+                .onItem().transform(rowCount -> {
+                    if (rowCount == 0) {
+                        return Response.status(Response.Status.NOT_FOUND).build();
+                    } else {
+                        return Response.ok().build();
+                    }
+                });
+    }
+
+    @PUT
+    @Path("/bundles/{bundleId}/behaviorGroups/{behaviorGroupId}/default")
+    public Uni<Response> setDefaultBehaviorGroup(@PathParam("bundleId") UUID bundleId, @PathParam("behaviorGroupId") UUID behaviorGroupId) {
+        return behaviorGroupResources.setDefaultBehaviorGroup(bundleId, behaviorGroupId)
+                .onItem().transform(rowCount -> {
+                    if (rowCount == 0) {
+                        return Response.status(Response.Status.NOT_FOUND).build();
```

Comment:
I think this would return `NOT_FOUND` if the `behaviorGroupId` is already the default behavior group.